### PR TITLE
People and Spaces pages not visible for guests

### PR DESCRIPTION
**Pull request by s-tyshchenko**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [x] When resolving a specific issue, it's referenced in the PR's description (e.g. `Fix #xxx[,#xxx]`, where "xxx" is the Github issue number)
- [x] All tests are passing
- [x] New/updated tests are included
- [x] Changelog was modified

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by s-tyshchenko** [comment]
Screenshot:
Снимок экрана 2022-03-06 в 13 29 28

File: protected/humhub/modules/admin/views/authentication/authentication.php [hunk 1]
```diff
@@ -53,3 +59,19 @@
     <?php ActiveForm::end(); ?>
 </div>
 <?php $this->endContent(); ?>
+
+<script <?= \humhub\libs\Html::nonce() ?>>
```
Comment (ArchBlood) [inline protected/humhub/modules/admin/views/authentication/authentication.php:63]: Question, is it necessary to run the script within the view file?

File: protected/humhub/modules/user/controllers/PeopleController.php [hunk 2]
```diff
@@ -8,11 +8,15 @@
 namespace humhub\modules\user\controllers;
 
 use humhub\components\access\ControllerAccess;
+use humhub\components\behaviors\AccessControl;
 use humhub\components\Controller;
+use humhub\modules\space\permissions\SpaceDirectoryAccess;
```
Comment (ArchBlood) [inline protected/humhub/modules/user/controllers/PeopleController.php:13]: Is `SpaceDirectoryAccess` needed?

File: protected/humhub/modules/user/controllers/PeopleController.php [hunk 3]
```diff
@@ -8,11 +8,15 @@
...
 use humhub\modules\user\components\PeopleQuery;
+use humhub\modules\user\helpers\AuthHelper;
 use humhub\modules\user\permissions\PeopleAccess;
 use humhub\modules\user\widgets\PeopleCard;use Yii;
```
Comment (ArchBlood) [inline protected/humhub/modules/user/controllers/PeopleController.php:17]: Not sure if it's a mobile issue or an actual issue, but from viewing it it seems that a line break was missed here.

**Review by ArchBlood — COMMENTED** [review]
Just dropping a few questions about the p/r

**Pull request closed without merging**